wrangler-service/src/main/java/co/cask/wrangler/service/directive/DirectivesService.java

```diff
@@ -672,6 +673,9 @@ public void execute(HttpServiceRequest request, HttpServiceResponder responder,
     } catch (DirectiveParseException e) {
       LOG.warn(e.getMessage(), e);
       error(responder, e.getMessage());
+    } catch (RecipeException e) {
+      LOG.error(e.getMessage(), e);
```

Comment:
can you open a jira to improve the logging in wrangler? Users errors should not be logged as service errors or warnings. A user running directives on invalid data is a normal operation.

Reply:
Filed: https://issues.cask.co/browse/CDAP-14393